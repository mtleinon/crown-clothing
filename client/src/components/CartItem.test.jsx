import { shallow } from 'enzyme';
import React from 'react';
import { CartItem } from './CartItem';

it('Expect to render CartItem component', () => {
  const mockItem = {
    imageUrl: 'url',
    name: 'test',
    quantity: 3,
    price: 123.32
  };
  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
