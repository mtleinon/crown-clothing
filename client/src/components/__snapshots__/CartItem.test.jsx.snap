// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render CartItem component 1`] = `ShallowWrapper {}`;
